package.json

```diff
@@ -75,7 +76,7 @@
     "@rollup/plugin-json": "^4.1.0",
     "@rollup/plugin-node-resolve": "^13.0.4",
     "@rollup/plugin-url": "^7.0.0",
-    "@schematics/angular": "~15.2.0",
+    "@schematics/angular": "~16.0.0-rc.3",
     "@storybook/addon-essentials": "^7.0.2",
     "@storybook/angular": "^7.0.2",
```

Comment:
7.1.0-alpha.11 is required for angular 16 support https://github.com/storybookjs/storybook/pull/22096#issuecomment-1530884317